quidel_covidtest/delphi_quidel_covidtest/export.py

```diff
@@ -1,39 +0,0 @@
-# -*- coding: utf-8 -*-
-"""Function to export the dataset in the format expected of the API.
-"""
-import numpy as np
-import pandas as pd
-
-def export_csv(df, geo_name, sensor, receiving_dir, start_date, end_date):
-    """Export data set in format expected for injestion by the API
-    Parameters
-    ----------
-    df: pd.DataFrame
-        data frame with columns "geo_id", "timestamp", and "val"
-    geo_name: str
-        name of the geographic region, such as "state" or "hrr"
-    sensor: str
-        name of the sensor; only used for naming the output file
-    smooth: bool
-        should the signal in "val" be smoothed?
-    receiving_dir: str
-        path to location where the output CSV files to be uploaded should be stored
-    start_date: datetime.datetime
-        The first date to report
-    end_date: datetime.datetime
-        The last date to report
-    """
-
-    df = df.copy()
-    df = df[np.logical_and(df["timestamp"] >= start_date,
-                           df["timestamp"] <= end_date)]
-
-    for date in df["timestamp"].unique():
-        t = pd.to_datetime(str(date))
-        date_short = t.strftime('%Y%m%d')
-        export_fn = f"{date_short}_{geo_name}_{sensor}.csv"
-        result_df = df[df["timestamp"] == date][["geo_id", "val", "se", "sample_size"]]
-        result_df = result_df[result_df["sample_size"].notnull()]
```

Comment:
This line doesn't appear to be in the util, is it ok to remove?

Reply:
hm not sure actually.  I was hoping you could refactor the null culling to be outside the call to `export_csv`, but that fails because it doesn't write (blank) files for dates with all null sample sizes.  Does this make a difference in practice?  When do we expect to have null values there?

Reply:
@krivard may have the better context.

Reply:
Sample sizes are null when any of the following are true:
* the signal is not based on sampling (as in cases/deaths)
* sampling is done by our data source, and is not exposed to us (as in ght/gs)
* sampling would reveal the market share of our data source, and as a courtesy to them we do not publish it (dv, ha, possibly quidel?)

Reply:
Is not having blank files for certain dates a problem? Worst case we could write a blank file creator, since `to_csv()` will still work on an empty df.

Reply:
Fixed.